src/cljs/ataru/virkailija/application/handlers.cljs

```diff
@@ -253,7 +253,11 @@
                                                      :option-answers           option-answers
                                                      :states-and-filters       {:attachment-states-to-include (get-in db [:application :attachment-state-filter])
                                                                                 :processing-states-to-include (get-in db [:application :processing-state-filter])
-                                                                                :filters                      (get-in db [:application :filters])
+                                                                                :filters                      (if (and search-term (not haku) (not form) (not hakukohde) (not hakukohderyhma))
```

Comment:
Tää ehto mua hiukan hämmentää.  Onko tarkoitus, että hakemusmaksurajain poistetaan käytöstä vain, jos hakutermi on annettu?  Tiketistä sain semmosen vaikutelman että myös ilman hakutermiä jäis rajaus pois, mutta mä saatan ymmärtää tuon tiketin väärin.

Reply:
eli siis hakemuksia ei lähdetä hakemaan ellei ole joko hakuehtoa tai hakua, etc valittuna

Reply:
Juu ok, ei haittaakaan.